index.js

```diff
@@ -64,6 +64,7 @@ module.exports = function(app) {
     });
 
     app.use(function (err, req, res) {
```

Comment:
I don't get it: the last change seems to revert the effect altogether. This handler has to take 4 parameters if you want it to be understood as an error handler.

> This is probably the only totally needed change in this PR, actually :wink: